builds/e2e/e2e.yaml

```diff
@@ -7,14 +7,12 @@ resources:
     source: 'Azure-IoT-Edge-Core Build Images'
     trigger:
       branches:
-      - master
-      - release/*
+      - release/1.1
```

Comment:
Master and 1.2 are configured like this:
https://github.com/Azure/iotedge/blob/master/builds/e2e/e2e.yaml#L12

I think what you actually want to change is this, not the trigger:
https://github.com/Azure/iotedge/blob/master/builds/e2e/e2e.yaml#L11

Reply:
https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema%2Cparameter-schema#pipeline-resource

Reply:
Agreed. I assume you're trying to change which artifacts are chosen by default in a manual or scheduled run of the pipeline (as opposed to a run triggered by commits landing in a branch)?